server/src/pages/shot/time-diff.js

```diff
@@ -24,6 +24,12 @@ exports.TimeDiff = class TimeDiff extends React.Component {
     }
   }
 
```

Comment:
@flod or @stasm - This file contains a number of functions that create partial time-related strings, like "in 1 day", or "3 hours ago". These partial strings are then concatenated with phrases like "expires " or "expired ", or, in the case of the `shotPageRestoreButton` l10n ID, "restore for 30 days". 

What's the best way to handle these relative times, given that some languages are declined, so that we couldn't simply concatenate strings to render "restore for 1 day" vs. "expires in 1 day" vs. "expired 1 day ago"?